docs/app/Components/ComponentDoc/ComponentExample/ComponentExample.js

```diff
@@ -206,7 +206,7 @@ class ComponentExample extends Component {
     const FAKER = require('faker')
     const LODASH = require('lodash')
     const REACT = require('react')
-    const SEMANTIC_UI_REACT = require('semantic-ui-react')
+    const STARDUST = require('stardust')
```

Comment:
Changing the name now will make it way more convenient to work with the docs and old library in parallel.